traitcuration/traits/tasks.py

```diff
@@ -26,3 +26,10 @@ def get_clinvar_data_and_suggestions():
     clinvar.run_clinvar()
     zooma.run_zooma_for_all_traits()
     return 'Successful Import'
+
+
+@shared_task
+def get_ols_status():
+    ols.check_term_status()
+    ols.check_awaiting_import_terms()
+    return 'Successful status update'
```

Comment:
I would suggest having only one call to some main function in the `ols` module here, which would do all necessary processing steps. Otherwise, if we have multiple top-level functions in that module and multiple calls here, it creates a possibility for code sync problems